README.md

```diff
@@ -216,6 +217,42 @@ If you use the `HttpBodyStream` the whole transfer will be [chunked encoded](htt
 
 Check out the `examples` folder how your computation could look like.
 
+### Streaming requests
+
+The client can send chunks of data to the server through the [chunked transfer encoding](https://en.wikipedia.org/wiki/Chunked_transfer_encoding).
```

Comment:
The changes LGTM :+1: Except for one thing: What if the client sends a 1GB upload request *without* using chunked transfer encoding?